module/state_synchronization/indexer/indexer_core.go

```diff
@@ -30,7 +31,7 @@ type IndexerCore struct {
 	collections  storage.Collections
 	transactions storage.Transactions
 	results      storage.LightTransactionResults
-	batcher      bstorage.BatchBuilder
+	batcher      *pebble.DB
```

Comment:
while we are at it, could we rename this to `database`. Given the following line, I have a hard time making sense of the naming here: 
https://github.com/onflow/flow-go/blob/51969ecf92ea9775622890d6be9be223f6f7a593/module/state_synchronization/indexer/indexer_core.go#L149